mico-core/src/main/java/io/github/ust/mico/core/persistence/MicoServiceRepository.java

```diff
@@ -43,6 +43,9 @@
 
     @Depth(2)
     Optional<MicoService> findByShortNameAndVersion(String shortName, String version);
+    
+    @Query("MATCH (application:MicoApplication)-[:INCLUDES]-(service:MicoService) WHERE application.shortName = {applicationShortName} AND application.version = {applicationVersion} RETURN COLLECT(service) AS services")
```

Comment:
I'm just wondering how this works. Where is the `INCLUDES` defined?

Reply:
Right, that is weird, it should be `INCLUDES_SERVICE`, which it is now.